scripts/googleAppScripts/README.MD

```diff
@@ -31,3 +58,5 @@ To manually trigger the script for testing purposes:
 1. Within the Apps Script editor, select the `testOnFormSubmit` function.
 2. Click the run button to execute the script.
 3. Verify that the GitHub issue has been created with the correct labels and information.
+
+**Note:** Replace 'YOUR_ACTUAL_GITHUB_TOKEN' with your actual GitHub token when you run the `setPersonalGithubToken` function.
```

Comment:
you already mentioned that above

Reply:
Better be safe than sorry. Feel free to delete it